### ISSUE TITLE
Add a donation link

### ISSUE THREAD

**Issue by kumy** [geokrety/geokrety-website#55]
Following mail conversation.

In order to collect eventual donations from our users, we have to:
- [x] Create a paypal account
- [x] Add a link on the website
- [x] Post a news

**Comment by kumy** [comment]
Account created using our standard email.

**Comment by kumy** [comment]
@filipsPL @boly38 What do you think? Any ideas on rewording?

# Short text
![screenshot from 2018-04-15 19-06-39](https://user-images.githubusercontent.com/176794/38781084-871c4ab6-40e0-11e8-8ef3-6e204abe8f82.png)

# General design
With a bit more text…
![screenshot from 2018-04-15 19-06-21](https://user-images.githubusercontent.com/176794/38781085-873e5494-40e0-11e8-9dc8-5aa22cbd716f.png)

**Comment by kumy** [comment]
The associated news.
![screenshot from 2018-04-15 19-49-54](https://user-images.githubusercontent.com/176794/38781484-3b59206c-40e6-11e8-8d49-e65ae383f303.png)

**Comment by boly38** [comment]
maybe reword to "is and will stay free & open source". but not sure :)
for the panel I like more the first short version. but ok in any case.

**Comment by kumy** [comment]
All done.
